src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -212,6 +212,58 @@ static void leggTilManuelleFilter(BoolQueryBuilder queryBuilder, Filtervalg filt
         if (filtervalg.harEnsligeForsorgereFilter() && filtervalg.getEnsligeForsorgere().contains(EnsligeForsorgere.OVERGANGSSTØNAD)) {
             queryBuilder.must(existsQuery("enslige_forsorgere_overgangsstonad"));
         }
+
+        if (filtervalg.harDinSituasjonSvar()) {
+            BoolQueryBuilder brukerensSituasjonSubQuery = boolQuery();
+            filtervalg.registreringstype.forEach(dinSituasjonSvar -> {
+                if (dinSituasjonSvar.equals(DinSituasjonSvarDto.INGEN_DATA)) {
```

Comment:
Is it necessary to use `.equals` when comparing to enum values?

Reply:
Will fix.